docs/antctl.md

```diff
@@ -204,6 +204,16 @@ antctl get appliedtogroup [NAME] [-o yaml]
 antctl get addressgroup [NAME] [-o yaml]
 ```
 
+NetworkPolicy, AppliedToGroup, and AddressGroup also supports `sort-by=''` option,
+which can be used to sort these resources on the basis of a particular field. Any
+valid json path can be passed as flag value, if no value is passed it sorts the list by name.
+
+```bash
+antctl get networkpolicy --sort-by='.metadata.name'
```

Comment:
I have a question about this, according to the code, if the sort flag is empty, it will sort by name for network policy, it's actually using `.sourceRef.name` instead of `.metadata.name`, so I am wondering if we should provide the `.sourceRef.name` instead of '.metadata.name' here or highlight the difference with other objects? It may confuse user when they get different output with `sort-by=''` and `sort-by='.metadata.name'`

Reply:
we can change the example, but in the output the metadata name will be printed and they can verify that it is sorted on the basis of that field, and by default it is sorted by the actual networkpolicy name.

Reply:
I agree with @luolanzone here. We should highlight the difference, or use `.metadata.name` consistently for all 3.

I think it's as simple as updating this document as follows:

NetworkPolicy, AppliedToGroup, and AddressGroup also supports `sort-by=''` option,
which can be used to sort these resources on the basis of a particular field. Any
valid json path can be passed as the flag value. If no value is passed it will use a default field
to sort results. For NetworkPolicy, the default field is the name of the source NetworkPolicy.
For AppliedToGroup and AddressGroup, the default field is the object name (which is a generated
UUID).

```bash
antctl get networkpolicy --sort-by='.sourceRef.name'
antctl get appliedtogroup --sort-by='.metadata.name'
antctl get addressgroup --sort-by='.metadata.name'
```